core-web/.husky/pre-commit

```diff
@@ -18,14 +18,29 @@ NC='\033[0m' # No Color
 
 check_sdk_client_affected() {
```

Comment:
We have a problem around this function.   It is using and expecting yarn to be updated and runs even when there are no frontend changes.  We already have a conditional for when there are core-web changes.  we should probably use the same logic.

Note that the affected modules the method reports on based upon its current configuration will be all affected modules compared to origin master I think.  If there are commits in the PR that modify front-end modules then subsequent commits to the branch will still show the same modules modified even if the current commit does not touch core-web.  We just need to ensure that this behaves as we want, if we wrap with the other conditional that checks for any changes to core-web, this check is specific to the current PR.

# Run fixes on staged files
if [ -n "$core_web_files_staged" ]; then
    perform_frontend_fixes "${core_web_files_staged}" true
    errors=$?  # Capture the return value from perform_frontend_fixes
fi
```